src/map/status.cpp

```diff
@@ -4204,12 +4405,32 @@ int status_calc_pc_sub(struct map_session_data* sd, enum e_status_calc_opt opt)
 			sd->subele[ELE_EARTH] += i;
 			sd->subele[ELE_FIRE] += i;
 			sd->subele[ELE_WIND] += i;
+#ifndef RENEWAL
 			sd->subele[ELE_POISON] += i;
 			sd->subele[ELE_HOLY] += i;
 			sd->subele[ELE_DARK] += i;
 			sd->subele[ELE_GHOST] += i;
 			sd->subele[ELE_UNDEAD] += i;
+#endif
 		}
+#ifdef RENEWAL
+		if (sc->data[SC_BASILICA]) {
+			i = sc->data[SC_BASILICA]->val1 * 5;
+			sd->right_weapon.addele[ELE_DARK] += i;
+			sd->right_weapon.addele[ELE_UNDEAD] += i;
+			sd->left_weapon.addele[ELE_DARK] += i;
+			sd->left_weapon.addele[ELE_UNDEAD] += i;
+			sd->magic_addele[ELE_HOLY] += sc->data[SC_BASILICA]->val1 * 3;
+		}
+		if (sc->data[SC_FIREWEAPON])
+			sd->magic_addele[ELE_FIRE] += sc->data[SC_FIREWEAPON]->val1;
+		if (sc->data[SC_WINDWEAPON])
+			sd->magic_addele[ELE_WIND] += sc->data[SC_WINDWEAPON]->val1;
+		if (sc->data[SC_WATERWEAPON])
+			sd->magic_addele[ELE_WATER] += sc->data[SC_WATERWEAPON]->val1;
+		if (sc->data[SC_EARTHWEAPON])
+			sd->magic_addele[ELE_EARTH] += sc->data[SC_EARTHWEAPON]->val1;
```

Comment:
```suggestion
			sd->magic_atk_ele[ELE_EARTH] += sc->data[SC_EARTHWEAPON]->val1;
```
when will github support multiline to remove in suggestion